### PR TITLE
[12.0][FIX] sale_milestone_profile_invoicing: apropriate comparing

### PR DESCRIPTION
**Pull request by i-vyshnevska**

### CODE REVIEW

**Review by simahawk — APPROVED** [review]
seems legit

**Review by TDu — APPROVED** [review]

File: sale_milestone_profile_invoicing/models/sale_order_line.py [hunk 1]
```diff
@@ -149,7 +153,7 @@ def _compute_invoice_status(self):
         for line in self:
             if line._is_linked_to_milestone_product():
                 mapping = line.project_id.sale_line_employee_ids.filtered(
-                    lambda r: r.sale_line_id == line.id
+                    lambda r: r.sale_line_id.id == line.id
```
Comment (pedrobaeza) [inline sale_milestone_profile_invoicing/models/sale_order_line.py]: Better to put

```suggestion
                    lambda r: r.sale_line_id == line
```

**Comment by i-vyshnevska** [comment]
@pedrobaeza  updated

**Pull request merged**